modules/insticatorBidAdapter.js

```diff
@@ -0,0 +1,272 @@
+import { config } from '../src/config.js';
+import { BANNER } from '../src/mediaTypes.js';
+import { registerBidder } from '../src/adapters/bidderFactory.js';
+import {
+  deepAccess,
+  generateUUID,
+  logError,
+} from '../src/utils.js';
+import { getStorageManager } from '../src/storageManager.js';
+
+export const storage = getStorageManager();
```

Comment:
If you have a GVL ID, which I think you do looking at your docs PR, you can pass that while initialising Storage Manager. That'll ensure your adapter doesn't gets blocked (when user has given consent) if [GDPR Enforcement](https://docs.prebid.org/dev-docs/modules/gdprEnforcement.html) module is used. You can refer `appnexus` to see a sample usage.